### PR TITLE
remove --protocol args

### PR DESCRIPTION
**Pull request by zhoney**
Change-Id: I48772554458ba4e9ac8ffc4e418a433b8a0aab20

### CODE REVIEW

File: src/main/java/com/baidu/hugegraph/base/ToolClient.java [hunk 1]
```diff
@@ -82,20 +82,18 @@ public ObjectMapper mapper() {
...
         private Integer timeout;
-        private String protocol;
         private String trustStoreFile;
         private String trustStorePassword;
 
         public ConnectionInfo(String url, String graph,
-                              String username, String password,
-                              Integer timeout, String protocol,
-                              String trustStoreFile, String trustStorePassword) {
+                              String username, String password, Integer timeout,
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/base/ToolClient.java]: wrap line after "String password,"

File: src/main/java/com/baidu/hugegraph/base/ToolClient.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public ToolClient(ConnectionInfo info) {
         this.client = HugeClient.builder(info.url, info.graph)
                                 .configUser(info.username, info.password)
                                 .configTimeout(info.timeout)
-                                .configSSL(info.protocol, info.trustStoreFile,
+                                .configSSL(info.trustStoreFile,
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/base/ToolClient.java]: set trustStoreFile to default value if info.url starts with 'https' and trustStoreFile is empty

**Review by javeme — CHANGES_REQUESTED** [review]

**Review by Linary — DISMISSED** [review]

File: src/main/java/com/baidu/hugegraph/base/ToolClient.java [hunk 3]
```diff
@@ -29,6 +29,9 @@
 
 public class ToolClient {
 
+    private static final String DEFAULT_TRUST_STORE_FILE = "./bin/keystore";
+    private static final String DEFAULT_TRUST_STORE_PASSWORD = "123456";
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/base/ToolClient.java]: set hugegraph

File: src/main/java/com/baidu/hugegraph/base/ToolClient.java [hunk 4]
```diff
@@ -29,6 +29,9 @@
 
 public class ToolClient {
 
+    private static final String DEFAULT_TRUST_STORE_FILE = "./bin/keystore";
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/base/ToolClient.java]: move to conf

**Review by javeme — CHANGES_REQUESTED** [review]

**Review by Linary — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**